build-tools/create-dotnet-pkg/create-dotnet-pkg.csproj

```diff
@@ -78,6 +85,10 @@
     <ItemGroup>
       <FileWrites Include="$(PkgProductOutputPath)" />
       <None Include="$(PkgProductOutputPath)" CopyToOutputDirectory="PreserveNewest" Link="Microsoft.NET.Workload.Android-$(AndroidPackVersionLong).pkg" />
+      <FilesToSign Include="$(OutputPath)\Microsoft.NET.Workload.Android-$(AndroidPackVersionLong).pkg">
```

Comment:
`$(OutputPath)` should always end with `\`, so this might be putting two `\\`.

Reply:
I am not sure if this is being added automatically, but it currently doesn't contain a trailing slash: https://github.com/xamarin/xamarin-android/blob/master/build-tools/create-dotnet-pkg/create-dotnet-pkg.csproj#L5

I could update the rest of the project file to do this but it's maybe not worth the diff?

Reply:
Ok maybe only Microsoft.NET.Sdk adds the `\` and not Microsoft.Build.NoTargets.

If you don't see `\\` in the log, this should be fine.